docs/python/1.6/python-appapi-devguide-2.md

```diff
@@ -10,7 +10,7 @@ prev:
   title: Developing IBM Streams Applications with Python
 next:
   file: python-appapi-devguide-2a
-  title: 2.0 Developing for the IBM Streaming Analytics service
+  title: 2.0 Tutorial: Developing for the IBM Streaming Analytics service
```

Comment:
The other dev guides don't aren't prefixed with `Tutorial:`. I think we should stick to the convention the others use, or change it for all of them.